packages/govern-contract-utils/package.json

```diff
@@ -22,11 +22,11 @@
     "hardhat": "^2.0.2"
   },
   "scripts": {
-    "build": "yarn compile && hardhat typechain",
+    "build": "yarn compile",
```

Comment:
What exactly was the reason to change those commands within this PR?

Reply:
The command hardhat type chain is not longer relevant as types are generated during compilation.